src/settings/RoutingAddress/RoutingAddress.js

```diff
@@ -17,7 +17,22 @@ import css from '../ConfigManagerForm.css';
 export const RoutingAddress = ({ label }) => {
   const showCallout = useShowCallout();
   const { addressTypes } = useUserAddressTypes();
-  const { data, isFetching, refetch } = useRoutingAddressSettings();
+
+  const { data, isLoading, refetch } = useRoutingAddressSettings({
+    onError: ({ response }) => {
+      if (response?.status === 403) {
```

Comment:
not clear how it's possible to get this error, when you included BE permissions to UI set and without UI permission it's not possible to open this setting at all, please remove useless code